enterprise/fulltext-addon/src/main/java/org/neo4j/kernel/api/impl/fulltext/FulltextProvider.java

```diff
@@ -220,6 +222,28 @@ public ReadOnlyFulltext getReader( String identifier, FulltextIndexType type ) t
         }
     }
 
+    public Set<String> getProperties( String identifier, FulltextIndexType type )
+    {
+        return applyToMatchingIndex( identifier, type, LuceneFulltext::getProperties );
+    }
+
+    private <E> E applyToMatchingIndex( String identifier, FulltextIndexType type, Function<LuceneFulltext,E> getProperties )
```

Comment:
This `getProperties` function should have a more generic name, I think.

Reply:
whoops